request/request.go

```diff
@@ -128,3 +128,19 @@ func GetExtraResources(req *v1beta1.RunFunctionRequest) (map[string][]resource.E
 	}
 	return out, nil
 }
+
+func GetCredentials(req *v1beta1.RunFunctionRequest) (map[string]resource.Credential, error) {
```

Comment:
```suggestion
func GetCredentials(req *v1beta1.RunFunctionRequest, name string) (resource.Credential, error) {
```

I wonder if this would be more convenient - i.e. to get specific named credentials rather than the whole map?

Reply:
Maybe also remove the `s` from `credentials`?

Reply:
Yes, good point. I think that's better as well

Reply:
Nit: my vote is for `Credentials`. Usually by convention even if it's a single credential (e.g. username and password) it's more common in tech to say credentials (plural) for some reason. Plus if `GetCredentials` returns credential data it is actually a map with potentially multiple keys - i.e. potentially multiple credentials.

https://english.stackexchange.com/questions/252073/which-is-correct-credential-or-credentials